starknet_devnet/starknet_wrapper.py

```diff
@@ -940,6 +949,9 @@ async def __predeclare_starknet_cli_account(self):
         """Predeclares the account class used by Starknet CLI"""
         state = self.get_state().state
         state.compiled_classes[STARKNET_CLI_ACCOUNT_CLASS_HASH] = oz_account_class
+        if self.config.verbose or not self.config.hide_predeployed_accounts:
```

Comment:
Should the logical operator be `and` or `or`?

Reply:
`or` because the verbose flag has a higher priority over hiding, meaning it discards all hiding flags. 
It was independent in the first commits, but you pointed out that behavior.